File: java-files-corrected/56.java
class ExceptionHandling {
    public static void main(String[] args) {
        try {
            int num = 10, den = 0;

            if (num == 0) {
                throw new IllegalArgumentException("Cannot divide by zero.");
            }

            System.out.println("Result: " + num / den);
        }
        catch (Exception e) {
            System.out.println("Error: " + e.getMessage());
        }
    }
}

File: cpp-test-files/56.cpp
//try-catch
#include <iostream>
#include <stdexcept>

int main() {
    try {
        int num = 10;
        int den = 0;

        if (den == 0) {
            throw std::invalid_argument("Cannot divide by zero.");
        }

        std::cout << "Result: " << num / den << std::endl;
    }
    catch (const std::exception& e) {
        std::cout << "Error: " << e.what() << std::endl;
    }

    return 0;
}